src/models/campaigns.model.js

```diff
@@ -17,7 +17,7 @@ function createModel(app) {
   const { Schema } = mongooseClient;
   const campaign = new Schema(
     {
-      title: { type: String, required: true },
+      title: { type: String, required: true, index: true },
```

Comment:
It should be a text index

Reply:
@aminlatifi 
You are right, I changed and added a test case ( before adding index the test was failing but after adding index the test passed)